sdl-core/src/main/scala/io/smartdatalake/util/misc/GenericSchemaUtil.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Smart Data Lake - Build your data lake the smart way.
+ *
+ * Copyright © 2019-2023 ELCA Informatique SA (<https://www.elca.ch>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package io.smartdatalake.util.misc
+
+import io.smartdatalake.definitions.Environment
+import io.smartdatalake.workflow.dataframe.GenericSchema
+import org.slf4j.LoggerFactory
+
+object GenericSchemaUtil {
+
+  def columnExists(schema: GenericSchema, colName: String): Boolean = {
+    if (Environment.caseSensitive) schema.columns.contains(colName)
+    else schema.columns.contains(colName.toLowerCase)
+  }
+
+  def filterColumns(schema: GenericSchema, columnsFilterList: Seq[String], includeColumns: Boolean = true): Seq[String] = {
```

Comment:
Would be nice to add some scaladoc for this function, explaining parameter includeColumns.

Reply:
Fixed in commit c92d31081886560b062c25d383f1d61eaeb02d90